source/opt/function.h

```diff
@@ -104,19 +104,26 @@ class Function {
     });
   }
 
-  // Runs the given function |f| on each instruction in this function, and
-  // optionally on debug line instructions that might precede them.
+  // Runs the given function |f| on instructions in this function, in order,
+  // and optionally on debug line instructions that might precede them.
+  // If |f| returns false, iteration is terminated.
```

Comment:
This is not true for the "ForEachInst" version.  |f| has no return value.

Reply:
Fixed in next commit